src/main/java/hudson/plugins/doxygen/DoxygenArchiver.java

```diff
@@ -5,12 +5,7 @@
 import hudson.Extension;
 import hudson.FilePath;
 import hudson.Launcher;
-import hudson.matrix.MatrixAggregatable;
-import hudson.matrix.MatrixAggregator;
-import hudson.matrix.MatrixConfiguration;
-import hudson.matrix.MatrixRun;
-import hudson.matrix.MatrixBuild;
-import hudson.matrix.MatrixProject;
+import hudson.matrix.*;
```

Comment:
Probably an IDE thing but generally explicit imports preferred if not test code.